src/SlackInterface.js

```diff
@@ -4,27 +4,133 @@
 
 /* eslint no-console:0 */
 
-var RtmClient = require('@slack/client').RtmClient;
-var RTM_EVENTS = require('@slack/client').RTM_EVENTS;
+/*******************************************************************************
+ * Globals
+ */
+const RtmClient = require('@slack/client').RtmClient;
+const RTM_EVENTS = require('@slack/client').RTM_EVENTS;
+const AWS = require('./AWS_API.js');
+var rtm;
+
+const token = process.env.SLACK_API_TOKEN || '';
+const DEBUG = process.env.DEBUG || false;
+AWS.DEBUG = DEBUG;
+
+
+/*******************************************************************************
+ * Functions
+ */
 
+var handleRtmMessage = function(message) {
+    if (DEBUG) { console.log('Message:', message) }
+
+    var text = message.text;
+
+    if (keyMessage(text, 'aws ')) {
```

Comment:
I actually did think about this, the problem is that how would you dynamically link these to the functions.

In this case I just tried to keep it so that constants are set in specific places so it's fairly easy to change them, but that would definitely be a good place for improvement.

Any ideas how we can do that?